src/analyzer/analyzersilence.cpp

```diff
@@ -9,18 +9,31 @@ constexpr float kSilenceThreshold = 0.001;
 // TODO: Change the above line to:
 //constexpr float kSilenceThreshold = db2ratio(-60.0f);
 
-}  // anonymous namespace
+bool shouldUpdateMainCue(CuePosition cue) {
+    return cue.getSource() != Cue::MANUAL || cue.getPosition() == -1.0 || cue.getPosition() == 0.0;
+}
+
+bool hasIntroCueStart(const Cue& cue) {
+    return cue.getPosition() != -1.0;
+}
+
+bool hasOutroCueEnd(const Cue& cue) {
```

Comment:
Same here: Can we omit outro in the function name?

Reply:
Same here. Let's keep it as is.